src/btree/bt_read.c

```diff
@@ -355,6 +355,8 @@ __page_read(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
         for (i = 0; i < count - 1; ++i)
             __wt_buf_free(session, &deltas[i]);
         WT_ERR(ret);
+
+        WT_STAT_CONN_DSRC_INCR(session, cache_read_delta);
```

Comment:
What is the intended meaning of this stat? This will only increment it by one, which doesn't feel correct when we might've read (say) 32 deltas. Would `WT_STAT_CON_DSRC_INCRV(session, cache_read_delta, count - 1)` be more appropriate?